scripts/asm-installer/install_asm

```diff
@@ -733,6 +748,10 @@ validate_args() {
       fatal "Citadel is not supported with managed control plane."
     fi
 
+    if [[ "${CA}" == "private_ca" ]]; then
+      fatal "Google Certificate Authority Service integration is not supported with managed control plane."
```

Comment:
Can we be more consistent with naming? The argument is `private_ca`, in the usage prompt it's `GCP CAS`, and here it's different and doesn't match GCP CAS.